tests/unit/test_cryptoSupport.c

```diff
@@ -17,7 +17,7 @@
 #include "ecdsa_privkey.h"
 #include "safe_lib.h"
 #include "fdotypes.h"
-#include "test_RSARoutines.h"
+//#include "test_RSARoutines.h"
```

Comment:
shouldn't this line be deleted?

Reply:
Updated in commit 32d6b292386138a719c01fb05eb81ef16ec7d2f6